### PR TITLE
Use rems for typographical properties

### PR DESCRIPTION
**Pull request by csswizardry**
Things got a bit messy in this issue—there are lots of reversions and stuff in here, so please make sure you give it a good looking over.

Current status:

* We’re keeping `_tools.rem.scss`
* We’re using that in `_generic.shared.scss`
* We are not using it on margins etc. in any objects.

### CODE REVIEW

File: example.main.scss [hunk 1]
```diff
@@ -23,6 +23,8 @@
  * Global...............Project-wide variables and settings.
  *
  * TOOLS
+ * Rem..................A function to convert a pixel input into a rem
```
Comment (nenadjelovac) [inline example.main.scss]: There's a mixin too. So something along the lines "A function and a mixin [...]" is more precise.

**Comment by nenadjelovac** [comment]
LGTM.

**Comment by csshugs** [comment]
LGTM.

**Comment by florianbouvot** [comment]
LGTM

**Pull request merged**